content/faq/lbry-app-basics

```diff
@@ -0,0 +1,127 @@
+---
+title: LBRY App Basic User Guide
+category: LBRY 101
+---
+
+Installing LBRY and setting it up can be a bit tough, especially if you encounter bugs or errors.
```

Comment:
Shouldn't start the document with saying installing is tough and then have the first topic be download/stream failing.  Maybe something more generic like, "Using LBRY is easy but sometimes you can run into bugs, here is a list of commonly asked questions and solutions."